synapse/tests/test_lib_stormlib_infosec.py

```diff
@@ -496,3 +502,75 @@ async def test_stormlib_infosec_vectToScore(self):
             for vect, norm in VECTORS_UNNORMAL:
                 valu = await core.callStorm(cmd, opts={'vars': {'vect': vect, 'vers': None}})
                 self.eq(norm, valu.get('normalized'))
+
+    async def test_stormlib_infosec_attack_flow(self):
```

Comment:
Would you also consider this the end-to-end test for the new handler logic?

Reply:
I was but I see your point. I'll add a separate test in the appropriate location. It'll probably look a lot like this though.